Functions/Backend/start.sh

```diff
@@ -49,8 +49,10 @@ FUNCTIONS_BIN="./node_modules/.bin/functions"
 "${FUNCTIONS_BIN}" deploy httpErrorTest --trigger-http
 "${FUNCTIONS_BIN}" deploy timeoutTest --trigger-http
 
-# Wait for the user to tell us to stop the server.
-echo "Functions emulator now running in ${TEMP_DIR}."
-read -n 1 -p "*** Press any key to stop the server. ***"
-echo "\nStopping the emulator..."
-"${FUNCTIONS_BIN}" stop
+if [ "$1" != "synchronous" ]; then
```

Comment:
Could you add a usage comment to the top of the script explaining the possible arguments and what they mean?

As it is, this is a bit confusing, mostly because it seems like an argument called "synchronous" should make the script block, but it actually makes it *not* block.